SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -328,6 +328,9 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Exposed local runtime IP address"))
 	FString ExposedRuntimeIP;
 
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Stop local deployment on stop play in editor"))
+	bool bStopSpatialOnEndPIE;
```

Comment:
```suggestion
	bool bStopDeploymentOnEndPIE;
```
Nit

Reply:
Maybe even specifying local to pair with `bAutoStartLocalDeployment`.

Reply:
I see that bStopSpatialOnExit is the inspiration here, but I feel like spatial is too wide a term. I initially thought the bStopSpatialOnExit would also shut down the spatial service, for example.

Reply:
Renamed to bStopLocalDeploymentOnEndPIE